specs/breadcrumb.md

```diff
@@ -0,0 +1,132 @@
+# Breadcrumb
+
+## Overview
+A `fast-breadcrumb` component is used as a navigational aid, allowing users to maintain awareness of their locations within a program, app, or a website.
+
+### Use Cases
+- James is browsing an ecommerce website and is looking for a few items to add to his new kitchen. He needs a knife set, and an espresso machine. He checks out `Home and Kitchen` page, then selects `Kitchen and Dining`, first category he picks is `Cutlery`, he finds the sets of knives. There is now a breadcrumb trail of the pages James has clicked through. `Home and Kitchen /  Kitchen and Dining / Cutlery / Knife Sets`
+He clicks on `Kitchen and Dining` to go back to the category of kitchen items, selects `Coffee and Espresso` finds the best one that suits his needs. If James wants to continue looking for another item for his kitchen, or other things for his home, he just needs to click on any of the links in the breadcrumb trail to continue his search.
+
+### Prior Art/Examples
+- [Ant Design](https://ant.design/components/breadcrumb/)
+- [Atlaskit](https://atlaskit.atlassian.com/packages/core/breadcrumbs)
+- [Carbon Design](https://www.carbondesignsystem.com/components/breadcrumb/code/)
+- [FAST Breadcrumb (React)](https://www.npmjs.com/package/@microsoft/fast-components-react-msft)
+- [Lightning Design System](https://www.lightningdesignsystem.com/components/breadcrumbs/#site-main-content)
+- [Material UI](https://material-ui.com/components/breadcrumbs/)
+- [Office Fabric](https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb)
+
+---
+
+### API
+
+*Component Name*
+- `fast-breadcrumb`
+
+### Anatomy and Appearance
+
+```ts
+<template
+    role="navigation"
+    aria-label="breadcrumb"
+>
+    <div role="list">
+        <slot></slot>
+    </div>
+</template>
+```
+
+*Slot Names*
+- default slot for items
+
+---
+
+## Implementation
+
+```html
+<fast-breadcrumb>
+    <fast-breadcrumb-item>
+        <fast-anchor appearance="lightweight">Home</fast-anchor>
+    </fast-breadcrumb-item>
+    <fast-breadcrumb-item>
+        <fast-anchor appearance="lightweight">Templates</fast-anchor>
+    </fast-breadcrumb-item>
+    <fast-breadcrumb-item aria-current="page">
+        <fast-anchor appearance="lightweight">Template Prices</fast-anchor>
+    </fast-breadcrumb-item>
```

Comment:
Is there a reason to have `fast-breadcrumb-item` if it's going to include fast anchors?

Reply:
`fast-anchor` is just a component you can use as a breadcrumb item. breadcrumb-item has a default slot to implement any control and also has the separator.

Reply:
In the current spec that doesn't seem to be the case; I need to slot fast-anchor into the fast-breadcrumb-item, which seems like an unnecessary abstraction here.

Reply:
I agree, does feel unnecessary. The issue I am currently running into is a way to inject the separator between each item and also have separator be any element. That is why we have breadcrumb-item. 
I am going to keep investigating this so we don't need a breadcrumb-item.